sidecar-injector/README.md

```diff
@@ -126,61 +242,272 @@ For this section, you'll work from a test namespace (test-namespace). Later you
-            sidecar-injector.secretless.io/config: "sleep-secretless-config"
+            sidecar-injector.cyberark.com/inject: "yes"
+            sidecar-injector.cyberark.com/secretlessConfig: "test-secretless-config"
+            sidecar-injector.cyberark.com/injectType: "secretless"
           labels:
-            app: sleep
+            app: test-app
         spec:
           containers:
-          - name: sleep
-            image: everpeace/curl-jq
-            command: ["/bin/sleep","infinity"]
+            - name: app
+              env:
+                - name: http_proxy
+                  value: "http://0.0.0.0:3000"
+              image: googlecontainer/echoserver:1.1
     EOF
     ```
 
-5. Verify sidecar container injected
+5. Verify Secretless sidecar container injected
     ```bash
     ~$ kubectl -n test-namespace get pods
     ```
     ```
     NAME                     READY     STATUS        RESTARTS   AGE
-    sleep-5c55f85f5c-tn2cs   2/2       Running       0          1m
+    test-app-5c55f85f5c-tn2cs   2/2       Running       0          1m
     ```
 
 6. Test Secretless
     ```bash
-    ~$ a_sleep_pod=$(kubectl \
+    ~$ a_test_pod=$(kubectl \
      -n test-namespace \
      get po \
-     -l=app=sleep \
+     -l=app=test-app \
      -o=jsonpath="{.items[0].metadata.name}")
     
     ~$ kubectl \
       -n test-namespace \
-      exec ${a_sleep_pod} \
-      -c sleep \
+      exec ${a_test_pod} \
+      -c app \
       -i \
       -- \
-      bash << 'EOL'
-    export http_proxy=localhost:8080
+      curl --silent localhost:8080 | grep authorization | sed -e s/^authorization=Basic\ // | base64 --decode; echo
+   
+    ```
+    ```
+    "test-secret#username:test-secret#password"
+    ```
     
-    response=$(curl --request GET --url http://scooterlabs.com/echo.json)
-    pretty_resp=$(echo "${response}" | jq -r .headers.Authorization)
-    echo '"'"$(echo "${pretty_resp##* }" | base64 --decode)"'"' | jq .
+
+## Conjur Authenticator/Secretless Sidecar Injection Example
+
+For this section, you'll work from a test namespace (sidecar-example-app). Later you will label this namespace with `cyberark-sidecar-injector=enabled` so as to allow the cyberark-sidecar-injector to operate on pods created in this namespace.
+
+1. Setup a Conjur appliance running with the Kubernetes authenticator installed and enabled - e.g. run `./start` in  [kubernetes-conjur-deploy](https://github.com/cyberark/kubernetes-conjur-deploy/)
+
+1. Load Conjur policy to create a host for the service account `$TEST_APP_SERVICE_ACCOUNT` - e.g. `test-app-secretless` and `test-app-secretless` are made available by walking through [kubernetes-conjur-demo](https://github.com/conjurdemos/kubernetes-conjur-demo) until `./3_init_conjur_cert_authority.sh`
+
+1. Set up environment variables modify to suite your needs e.g. use the same values from [kubernetes-conjur-demo](https://github.com/conjurdemos/kubernetes-conjur-demo)
+    ```bash
+    # required values
```

Comment:
Maybe separate these env vars into ones that are already defined if you go through `kubernetes-conjur-demo` and ones that aren't? They're all mixed up right now which made copy-pasting difficult.